File: mo-gateway/src/main/java/ink/ckx/mo/gateway/MoGatewayApplication.java
package ink.ckx.mo.gateway;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;

/**
 * @author chenkaixin
 * @description
 * @since 2023/11/11
 */
@EnableDiscoveryClient
@SpringBootApplication
public class MoGatewayApplication {

    public static void main(String[] args) {
        SpringApplication.run(MoGatewayApplication.class, args);
    }
}

File: mo-gateway/src/main/kotlin/ink/ckx/mo/gateway/MoGatewayApplication.kt
package ink.ckx.mo.gateway

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient

/**
 * @author chenkaixin
 * @description
 * @since 2023/11/11
 */
@EnableDiscoveryClient
@SpringBootApplication
class MoGatewayApplication

fun main(args: Array<String>) {
    runApplication<MoGatewayApplication>(*args)
}